### ISSUE TITLE
Login Screen Volume Slider

### ISSUE THREAD

**Issue by robmonte** [runelite/runelite#14133]
At some point in Runescape HD/RS3 history, Jagex added a volume slider to the login screen. They acknowledged that there were those that don't want to mute the music outright but couldn't tolerate how loud it was.

Currently in OSRS, it can only be turned off or be on at max volume. It would be nice if mousing over the mute/unmute icon, it then displayed a volume slider to adjust it. (Perhaps this could tie in with the request for custom login screen music? https://github.com/runelite/runelite/issues/11747)

**Comment by JBerben** [comment]
Would this be its own plugin? Or do you think it should be added as a feature to an existing plugin?
If I added this as a feature to an existing plugin, which plugin do you think should be changed? Music? Login screen? I just want to make sure that if this is added, it will be done as cleanly as possible.

**Comment by robmonte** [comment]
@JBerben Yeah that's a good question. At first I would think the music plugin would be the appropriate place for it, however things specific to the login screen may best be grouped within the login screen plugin.

Particularly so if there is a plugin feature to customize which song plays on the login screen (which I think would reside within the login screen plugin as well). Then I think it makes more sense for the volume control option to be there too so they are together.
